scheduler/roles/k8s_start_services/tasks/deploy_k8s_services.yml

```diff
@@ -85,6 +85,16 @@
     -p='[{"op": "replace", "path": "/spec/template/spec/containers/0/imagePullPolicy", "value":"IfNotPresent"}]'
   changed_when: true
 
+# HABANA PLUGIN
+- name: Deploy Habana Device plugin
+  ansible.builtin.command: "kubectl create -f '{{ habana_device_plugin_yaml_url }}'"
```

Comment:
habana device plugin should be deployed on nodes having gaudi only and not on the other nodes.

Eg: Let us say k8s cluster has 4 nodes out of which two nodes are having gaudi. In this case this plugin should be scheduled on 2 nodes having gaudi but not on other 2 nodes.

Reply:
The device plugin is a daemonset. More info: https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/

Reply:
What will be status of habana-device-plugin pod on the node which is not having gaudi?

Reply:
@francares Does the daemonset make use of a selector to ensure that it only runs on nodes with Gaudi?